File: targetIndicies.js
const targetIndices = function(nums, target) {
    const arr = [];
    let index = 0;

    for(let i=0; i<nums.length; i++){
      let min = i;
      for(let j=0; j<nums.length; j++){
        if(nums[i]<nums[j]){ 
          let tmp = nums[i];
          nums[i] = nums[j];
          nums[j] = tmp;
        }
      }
      
      
      
      if(i === nums.length-1){
        for(const numbers of nums ){
          if(numbers === target){
            arr.push(index)
          }
          index++;
        }
      }
    }

    return arr;
};

File: targetIndicies.py
def targetIndecies(nums, target):
    targetInd = []
    for i in range(len(nums)):
        for j in range(len(nums)):
            if (nums[i] < nums[j]):
                swap = nums[i]
                nums[i] = nums[j]
                nums[j] = swap

    for i in range(len(nums)):
        if nums[i] == target:
            targetInd.append(i)

    print(targetInd)
